<<<<<<<< HEAD:patches/monero/0001-fix-missing-___clear_cache-when-targetting-iOS.patch
From 6e284a2ef552f1f47e8ca9edcf8651312c9e37dd Mon Sep 17 00:00:00 2001
From: Czarek Nakamoto <cyjan@mrcyjanek.net>
Date: Tue, 2 Apr 2024 16:51:56 +0200
Subject: [PATCH 01/14] fix missing ___clear_cache when targetting iOS
========
From c7b6e26d36d2428e5747e2b8287ae3719fa443e3 Mon Sep 17 00:00:00 2001
From: Czarek Nakamoto <cyjan@mrcyjanek.net>
Date: Tue, 2 Apr 2024 16:51:56 +0200
Subject: [PATCH 02/14] fix missing ___clear_cache when targetting iOS
>>>>>>>> e3d763b (cleanup patches (and other stuff) (#79)):patches/monero/0002-fix-missing-___clear_cache-when-targetting-iOS.patch

---
 .gitmodules      | 3 ++-
 external/randomx | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
<<<<<<<< HEAD:patches/monero/0001-fix-missing-___clear_cache-when-targetting-iOS.patch
index 721cce3b4..ffb73fe9a 100644
========
index 721cce3..ffb73fe 100644
>>>>>>>> e3d763b (cleanup patches (and other stuff) (#79)):patches/monero/0002-fix-missing-___clear_cache-when-targetting-iOS.patch
--- a/.gitmodules
+++ b/.gitmodules
@@ -9,7 +9,8 @@
 	url = https://github.com/trezor/trezor-common.git
 [submodule "external/randomx"]
 	path = external/randomx
-	url = https://github.com/tevador/RandomX
+	url = https://github.com/MrCyjaneK/RandomX
+	branch = cyjan-fix-ios
 [submodule "external/supercop"]
 	path = external/supercop
 	url = https://github.com/monero-project/supercop
diff --git a/external/randomx b/external/randomx
index 102f8ac..ce72c9b 160000
--- a/external/randomx
+++ b/external/randomx
@@ -1 +1 @@
-Subproject commit 102f8acf90a7649ada410de5499a7ec62e49e1da
+Subproject commit ce72c9bb9cb799e0d9171094b9abb009e04c5bfc
-- 
<<<<<<<< HEAD:patches/monero/0001-fix-missing-___clear_cache-when-targetting-iOS.patch
2.48.1
========
2.39.5 (Apple Git-154)
>>>>>>>> e3d763b (cleanup patches (and other stuff) (#79)):patches/monero/0002-fix-missing-___clear_cache-when-targetting-iOS.patch

